internal/provider/resource_gitlab_project_membership.go

```diff
@@ -153,7 +163,11 @@ func resourceGitlabProjectMembershipSetToState(d *schema.ResourceData, projectMe
 	d.Set("project_id", projectId)
 	d.Set("user_id", projectMember.ID)
 	d.Set("access_level", accessLevelValueToName[projectMember.AccessLevel])
-
+	if projectMember.ExpiresAt != nil {
+		d.Set("expires_at", projectMember.ExpiresAt.String())
+	} else {
+	         d.Set("expires_at", "")
```

Comment:
Indentation doesn't match [here](https://github.com/gitlabhq/terraform-provider-gitlab/runs/6807171411?check_suite_focus=true) - Unfortunately, that came from my suggestion for which I didn't use proper tabs. Sorry, for that.

```suggestion
		d.Set("expires_at", "")
```

Reply:
my bad, I forgot to run the `make reviewable` command :'(
Ok I will apply your suggestions in another PR then !